### ISSUE TITLE
Please consider releasing source code alongside documentation on GitHub

### ISSUE THREAD

**Issue by headmin** [redcanaryco/mac-monitor#18]
**At a high level -- can you summarize your request?**

@Brandon7CC  - a kind request you to consider publishing the source code of your software along with the documentation. Many understand that your software is the product of your hard work and intellectual property. However, many believe that releasing the source code will be beneficial to the community, especially when you present your tool here on GitHub.

Your work has likely been inspired by other tools available in the community, such as https://github.com/objective-see and https://github.com/SuprHackerSteve/Crescendo. The community and stargazers already appreciate the effort you have put into developing your tool and encourages you to share the code along with the documentation.

In the #macadmins and security-conscious community, such transparency is highly valued, and having access to source code provides an opportunity to learn, collaborate, and improve for everyone.

**Comment by Brandon7CC** [comment]
Hey @headmin! This is not something that we're planning on doing. We're happy to write about technicalities at the request of the community though! 

Both of those communities have actually inspired us! This tool is an extension of our understanding of macOS internals + behavioral detection capabilities. The work of Steve Davis and Patrick Wardle also inspired parts of this tool 😄 a huge thank you goes out to them both. Please see #2 and the [Can you open source Mac Monitor](https://github.com/redcanaryco/mac-monitor#can-you-open-source-mac-monitor)?

**Comment by mgraeber-rc** [comment]
Hey! I just wanted to offer a heads up that @Brandon7CC just open-sourced [AtomicESClient](https://github.com/redcanaryco/mac-monitor/tree/main/AtomicESClient) for those wanting to learn ES/Swift. I hope you enjoy and can benefit from Brandon's great work!
